test/distributed/tensor/test_utils.py

```diff
@@ -19,6 +22,80 @@
 c10d_functional = torch.ops.c10d_functional
 
 
+class LocalTest(TestCase):
+    def test_explicit_order_placements(self):
+        # mesh_shape: ShapeType, placements: Sequence[Placement]
+        test_cases = [
+            {
+                "mesh_shape": [2, 2],
+                "placements": [Replicate()],
+                "ordered": [(0, Replicate())],
+            },
+            {
+                "mesh_shape": [2, 2],
+                "placements": [Shard(0)],
+                "ordered": [(0, Shard(0))],
+            },
+            {
+                "mesh_shape": [2, 2],
```

Comment:
can you make the `mesh_shape` have different mesh dim size instead, i.e. [2, 4] or [4, 2] to test the correctness